mdata/store_cassandra.go

```diff
@@ -362,7 +366,12 @@ func (o asc) Less(i, j int) bool { return o[i].sortKey < o[j].sortKey }
 
 func (c *CassandraStore) processReadQueue() {
 	for crr := range c.readQueue {
-		cassGetWaitDuration.Value(time.Since(crr.timestamp))
+		waitDuration := time.Since(crr.timestamp)
+		cassGetWaitDuration.Value(waitDuration)
+		if waitDuration > c.omitReadTimeout {
```

Comment:
this only drops reqRequests that have been in the queue too long.  The requests might have spent many minutes waiting to  to be added to c.readQueue due to it being full